assembl/static2/js/app/routes.jsx

```diff
@@ -84,12 +84,12 @@ const AdminChild = (props) => {
   case 'discussion':
     return <DiscussionAdmin {...props} section={props.location.query.section} />;
   case 'survey':
-    return <SurveyAdmin {...props} thematicId={props.location.query.thematic} section={props.location.query.section} />;
+    return <SurveyAdmin {...props} thematicId={props.location.query.thematicId} section={props.location.query.section} />;
   case 'brightMirror':
     return <BrightMirrorAdmin {...props} />;
   case 'thread':
     return <ThreadAdmin {...props} section={props.location.query.section} />;
-  case 'multiColumn':
+  case 'multiColumns':
```

Comment:
Not sure if it's necessary as it's admin page, but want to swap these switch statement strings with the constants value instead?